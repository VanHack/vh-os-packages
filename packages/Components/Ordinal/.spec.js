import React from 'react';
import VHOrdinal from '.';
import { render, renderer } from '../../../setupTests';

describe('[Component] - VHOrdinal', () => {
  it('renders component without crashing', () => {
    const { getByText } = render(<VHOrdinal value={1} />);
    expect(getByText('1st')).toMatchSnapshot();
  });

  it('renders VHOrdinal for 1', () => {
    const json = renderer(<VHOrdinal value={1} />).toJSON();
    const value = json.children[0];
    expect(value).toBe('1st')
  });

  it('renders VHOrdinal for 2', () => {
    const json = renderer(<VHOrdinal value={2} />).toJSON();
    const value = json.children[0];
    expect(value).toBe('2nd')
  });

  it('renders VHOrdinal for 3', () => {
    const json = renderer(<VHOrdinal value={3} />).toJSON();
    const value = json.children[0];
    expect(value).toBe('3rd')
  });

  it('renders VHOrdinal for 4', () => {
    const json = renderer(<VHOrdinal value={4} />).toJSON();
    const value = json.children[0];
    expect(value).toBe('4th')
  });
});
