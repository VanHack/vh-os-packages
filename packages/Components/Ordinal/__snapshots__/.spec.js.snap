// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] - VHOrdinal renders component without crashing 1`] = `
.c0 {
  width: 30px;
  font-size: 14px;
  color: #444;
  margin-right: 10px;
}

<span
  class="c0"
>
  1st
</span>
`;
